tdrs-frontend/src/mirage.js

```diff
@@ -8,11 +14,25 @@ export default function startMirage(
     environment,
 
     routes() {
-      this.namespace = 'mock_api'
+      this.urlPrefix = 'http://localhost:8080/v1'
+      this.post('/reports/', () => 'Success')
+      this.get('/auth_check/', () => {
+        if (
+          window.localStorage.getItem('loggedIn') ||
+          process.env.REACT_APP_PA11Y_TEST
+        ) {
+          return AUTH_CHECK_DATA
+        } else {
+          return FAILED_AUTH_CHECK_DATA
+        }
+      })
+      // if/when we add cypress tests, the rest of these routes will need some work done on them
 
-      this.post('/reports/', () => {
-        return 'Success'
+      this.patch('/users/set_profile', () => {
```

Comment:
We needed to mock this api so the frontend would not crash. If we want to have a more comprehensive mock, this should change some variables in local storage. Noting for future reference